docs/podman-build.1.md

```diff
@@ -39,15 +45,25 @@ instructions read from the Dockerfiles in the same way that environment
 variables are, but which will not be added to environment variable list in the
 resulting image's configuration.
 
+**--cache-from**
+
+Images to utilise as potential cache sources. Buildah does not currently support caching so this is a NOOP.
```

Comment:
utlise/utilize?  Did your spell checker spend too much time watching the Royal Wedding?

Reply:
My firefox has been in England mode for a while, not sure how to get it out of it. But this is a cut/paste from buildah bud man page, which has the english spelling.
Fixed.